### PR TITLE
Feature/sprint95 pinned message

### PR DESCRIPTION
**Pull request by Thrashattack**
Adds fetch channel pin message to chat sdk

### CODE REVIEW

**Review by cesarwbr — CHANGES_REQUESTED** [review]
It's missing the tests for these two files

**Review by fboechats — APPROVED** [review]

**Review by cesarwbr — APPROVED** [review]
ok

File: packages/sdk/src/channel/channel.ts [hunk 1]
```diff
@@ -442,7 +442,32 @@ export class Channel implements BaseChannel {
...
+   */
+  public async fetchPinMessage(): Promise<ChatMessage> {
+    if (this.sdk.site === null) {
+      throw new Error('Cannot fetch pinned message without a site id');
+    }
+    if (this.sdk.user === null) {
+      throw new Error('Cannot fetch pinned message without a user');
+    }
+    try {
+      const { GraphQLAPI } = await import('../services/graphql-api');
```
Comment (cesarwbr) [inline packages/sdk/src/channel/channel.ts:456]: you should use the current graphql instance `this.graphQLAPI`

File: packages/sdk/src/channel/channel.ts [hunk 2]
```diff
@@ -442,7 +442,32 @@ export class Channel implements BaseChannel {
...
   /**
-   * Remove message received listener
+   * Fetch Pin Messages for current channel
+   *
+   */
+  public async fetchPinMessage(): Promise<ChatMessage> {
+    if (this.sdk.site === null) {
+      throw new Error('Cannot fetch pinned message without a site id');
+    }
+    if (this.sdk.user === null) {
```
Comment (cesarwbr) [inline packages/sdk/src/channel/channel.ts:452]: the pin message should be available for anonymous user

**Review by cesarwbr — CHANGES_REQUESTED** [review]
check the messages

**Review by cesarwbr — APPROVED** [review]
ok

**Pull request merged**